changelog/db.changelog-1.5.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
jus what John mentioned tht I imagine `main` will have `1.5` innit by the time this is merged -- just need to keep on-top of it

Reply:
No 1.5 in main yet